### PR TITLE
Mailer

### PR DESCRIPTION
**Pull request by gonziesc**
travis
## Summary
- mailer created

### CODE REVIEW

**Comment by gonziesc** [comment]
Only check mailer commit

**Comment by mishuagopian** [comment]
@gonziesc but why do we have all these commits? Please fix that!

File: app/services/users.js [hunk 1]
```diff
@@ -8,6 +9,24 @@ exports.create = (user) => {
   });
 };
 
+exports.sendEmail = (user) => {
+  const transporter = nodemailer.createTransport({
```
Comment (mishuagopian) [inline app/services/users.js]: You should have a mailer service that exposes some function with params like to, subject and text. User and password should be set with the config and env vars.

**Comment by gonziesc** [comment]
It was because of the merge! fixed.

**Pull request merged**